decidim-conferences/lib/decidim/conferences/admin_engine.rb

```diff
@@ -97,113 +97,131 @@ class AdminEngine < ::Rails::Engine
+                            allowed_to?(:read, :attachment, conference: current_participatory_space),
+                        submenu: { target_menu: :conferences_admin_attachments_menu }
+          menu.add_item :conference_media_links,
+                        I18n.t("media_links", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.conference_media_links_path(current_participatory_space),
+                        if: allowed_to?(:read, :media_link, conference: current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.conference_media_links_path(current_participatory_space))
+          menu.add_item :conference_partners,
+                        I18n.t("partners", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.conference_partners_path(current_participatory_space),
+                        if: allowed_to?(:read, :partner, conference: current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.conference_partners_path(current_participatory_space))
+          menu.add_item :conference_speakers,
+                        I18n.t("conference_speakers", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.conference_speakers_path(current_participatory_space),
+                        if: allowed_to?(:read, :conference_speaker, conference: current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.conference_speakers_path(current_participatory_space))
+
+          menu.add_item :registrations,
+                        I18n.t("registrations", scope: "decidim.admin.menu.conferences_submenu"),
+                        "#",
+                        active: false, # is_active_link?(decidim_admin_conferences.conference_attachment_collections_path(current_participatory_space)),
+                        if: allowed_to?(:read, :conference_invite, conference: current_participatory_space) ||
+                            allowed_to?(:read, :registration_type, conference: current_participatory_space) ||
+                            allowed_to?(:read, :conference_registration, conference: current_participatory_space),
+                        submenu: { target_menu: :conferences_admin_registrations_menu }
+          menu.add_item :conference_user_roles,
+                        I18n.t("conference_admins", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.conference_user_roles_path(current_participatory_space),
+                        if: allowed_to?(:read, :conference_user_role, conference: current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.conference_user_roles_path(current_participatory_space))
+          menu.add_item :moderations,
```

Comment:
```suggestion

          menu.add_item :moderations,
```

Reply:
my understanding is that you requested to add a new line.